packages/web3-providers-ws/test/unit/supportsSubscriptions.test.ts

```diff
@@ -2,7 +2,7 @@ import Web3ProviderWS from '../../src/index';
 import { WebSocketOptions } from '../../src/types';
 
 describe('web3ProvidersWS.supportsSubscriptions', () => {
-    it('should return true', () => {
+    it('should return true', async () => {
         const web3ProvidersWS = new Web3ProviderWS('ws://127.0.0.1:8546');
         expect(web3ProvidersWS.supportsSubscriptions()).toBeTruthy();
```

Comment:
Please avoid using `toBeTruthy` matcher, and always match the exact type value the function is supposed to return.